src/plots/geo/layout_attributes.js

```diff
@@ -174,9 +174,31 @@ var attrs = module.exports = overrideAll({
             min: 0,
             dflt: 1,
             description: [
-                'Zooms in or out on the map view.',
-                'A scale of *1* corresponds to the largest zoom level',
-                'that fits the map\'s lon and lat ranges. '
+                'Initial zoom level of the map view.',
+                'A scale of *1* (100%) corresponds to the base zoom level',
+                'that fits the map\'s lon and lat ranges.'
+            ].join(' ')
+        },
+        minScale: {
+            valType: 'number',
+            role: 'info',
+            min: 0,
+            dflt: 0,
+            description: [
+                'Minimal zoom level of the map view.',
+                'A minScale of *0.5* (50%) corresponds to a zoom level',
+                'where the map has half the size of base zoom level.'
+            ].join(' ')
+        },
+        maxScale: {
+            valType: 'number',
+            role: 'info',
+            min: 0,
+            dflt: Infinity,
```

Comment:
`Infinity` is not a valid JSON value.
So maybe we should use `null` instead.

Reply:
fixed with c9f02d84746ae85d9f45b6fb7bd70996af03af55